var/spack/repos/builtin/packages/sendme/package.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```
Drop these two lines to follow the common package license header format. Otherwise this PR looks good to me!